packages/arb-token-bridge-ui/src/util/wagmi/setup.ts

```diff
@@ -19,10 +17,15 @@ import {
 import { isTestingEnvironment } from '../CommonUtils'
 import { ChainId } from '../networks'
 import { getCustomChainsFromLocalStorage } from '../networks'
+import { orbitChains } from '../orbitChainsList'
+import { getWagmiChain } from './getWagmiChain'
 
 const customChains = getCustomChainsFromLocalStorage().map(chain =>
   chainToWagmiChain(chain)
 )
+const wagmiOrbitChains = Object.keys(orbitChains).map(chainId =>
+  getWagmiChain(Number(chainId))
+)
```

Comment:
I think we can just call `getWagmiChain` always (it's just gonna call `chainToWagmiChain` internally anyway)

```suggestion
const wagmiCustomChains = getCustomChainsFromLocalStorage().map(chain =>
  getWagmiChain(chain)
)
const wagmiOrbitChains = getOrbitChains().map(chain => getWagmiChain(chain))
```

Reply:
done